### ISSUE TITLE
Home Assistant Cfg not working for driver based config

### ISSUE THREAD

**Issue by botioni** [openshwprojects/OpenBK7231T_App#406]
Home Assistant Cfg not working for driver based configuration and home assistant discovery also not working

**Comment by mariusbach** [comment]
It's really unhelpful to lack so many details. Device, scenario, drivers, config?

Are you using MQTT at all?

**Comment by botioni** [comment]
Yes im using mqtt . I meant if you are using driver config in pin asigment then it is not generating HA  config.

**Comment by GravityRZ** [comment]
i think you get a popup which states that it can not find a PWM, relay or something else

when you add a relay to the config and press the discovery button it works with me
so of we remove the popup block it should work right

**Comment by botioni** [comment]
Yup right if you add driver for the if off the pop-up it sould work

**Comment by iprak** [comment]
Driver based RGB should work, please add more details.

**Comment by botioni** [comment]
HA cfg generator for driver based not working what more details i can tel you set a driver based config with flag 4 and se if you can generate HA config

**Comment by iprak** [comment]
Are you using Discovery or copying the generate configuration? Are you getting the popup mentioned here: https://github.com/openshwprojects/OpenBK7231T_App/issues/416#issuecomment-1304529251 ?

**Comment by botioni** [comment]
I dont get config and I cant use auto both are not working and yes I'm
geting the pop-up

**Comment by iprak** [comment]
Ok, the popup issue was recently addressed in [1.14.129](https://github.com/openshwprojects/OpenBK7231T_App/releases/tag/1.14.129)

**Comment by GravityRZ** [comment]
@biotioni 
do you mean that when you go home assistant discovery the screen is empty(no code  visible)
or that you can not use discovery because you get the popup'(which is fixed in version 129)
